provisioning/e2e/_provisioning_e2e.js

```diff
@@ -130,99 +133,81 @@ var X509Group = function() {
   this.transports = [ Http ];
   var id = uuid.v4();
 
-  this._deviceId = 'deleteme_provisioning_node_e2e_' + id;
   this._registrationId = 'reg-' + id;
-  this._groupId = 'group-' + id;
+  this._groupId = 'testgroup';
 
   this.initialize = function(callback) {
-    debug('creating CA cert');
-    var caConfig = [
-      '[req]',
-      'req_extensions = v3_req',
-      'distinguished_name = req_distinguished_name',
-      '[req_distinguished_name]',
-      'commonName = IoTHub Test CA',
-      'commonName_max = 64',
-      '[v3_req]',
-      'basicConstraints = critical,CA:TRUE'
-    ].join('\n');
-    pem.createCertificate({ config: caConfig, commonName: 'IoTHub Test CA' }, function (err, caCert) {
+    debug('creating device cert');
+    self._factoryCert = {
+        cert: new Buffer(process.env.IOTHUB_CA_ROOT_CERT,"base64").toString(),
```

Comment:
>) [](start = 76, length = 1)

'ascii' ?

Reply:
huh.  utf-8 is default.  It should be the same thing, but I'll be explicit.

---
In reply to: [154233237](https://github.com/Azure/azure-iot-sdk-node/pull/166#discussion_r154233237) [](ancestors = 154233237)